webpack.config.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+const CleanWebpackPlugin  = require('clean-webpack-plugin'),
+  HtmlWebPackPlugin       = require('html-webpack-plugin'),
+  MiniCssExtractPlugin    = require('mini-css-extract-plugin'),
+  OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin'),
+  Package                 = require('./package.json'),
+  path                    = require('path'),
+  project_config          = require('./config.js'),
+  UglifyJsPlugin          = require('uglifyjs-webpack-plugin');
+
+const versionJS        = Package.webpack_bundle_version_js,
+      versionCSS       = Package.webpack_bundle_version_css,
+      buildDestination = './dist';
+
+// project_config overwrites default_config, edit project_config to customize the values below or add new fields
+const default_config = {
+  site: {
+    title: "MLH",
+    description: "The official collegiate hackathon league.",
+    baseurl: "/",
+    favicon_url: "img/favicon.ico"
+  }
+}
+
+const site_config = {...default_config, ...project_config};
+
+function generateHtmlPlugins (templateDir) {
+  const fs = require('fs');
+  const templateFiles = fs.readdirSync(path.resolve(__dirname, templateDir))
+
+  return templateFiles
+    .filter(item => item.includes('.hbs'))
+    .map(item => {
+      const parts = item.split('.')
+        const name = parts[0]
+        const extension = parts[1]
+        return new HtmlWebPackPlugin({
+          template: path.resolve(__dirname, `${templateDir}/${name}.${extension}`),
+          filename: `${name}.html`,
+          mobile: true,
```

Comment:
What is `mobile`?

Reply:
@theycallmeswift it adds meta tag in header for appropriate mobile scaling